Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -53,10 +71,29 @@ public class ConnectionPool {
     private let timeoutNs: Int64
     private let timeout: Int
     
-    /// Creates an instance of `ConnectionPool` containing `ConnectionPoolOptions.initialCapacity` connections.
-    /// The `connectionGenerator` will be invoked `ConnectionPoolOptions.initialCapacity` times to fill
-    /// the pool to the initial capacity.
-    ///
+    // MARK: Initializer
+    /**
+     Creates an instance of `ConnectionPool` containing `ConnectionPoolOptions.initialCapacity` connections.
+     The `connectionGenerator` will be invoked `ConnectionPoolOptions.initialCapacity` times to fill
+     the pool to the initial capacity.
+     ### Usage Example: ###
+     In this example, a closure to generate and release `Connection` instances are defined.
+     A connection pool is then initialized with given options, the `connectionGenerator` and the `connectionReleaser`.
+     ```swift
+     let options = ConnectionPoolOptions(initialCapacity: 2, maxCapacity: Int = 5, timeout: Int = 1000)
```

Comment:
`maxCapacity: Int = 5, timeout: Int = 1000` needs to be `maxCapacity: 5, timeout: 1000` or we get compilation failures